pkg/vm/iterator_test.go

```diff
@@ -0,0 +1,121 @@
+package vm
+
+import (
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/nspcc-dev/neo-go/internal/random"
+	"github.com/nspcc-dev/neo-go/pkg/core/interop/interopnames"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/callflag"
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+	"github.com/stretchr/testify/require"
+)
+
+type arrayIterator struct {
+	index  int
+	values []stackitem.Item
+}
+
+func TestCreateCallAndUnwrapIteratorScript(t *testing.T) {
```

Comment:
Oh, OK, I see that you have a separate test for it. Are you sure that `vm` is a proper place for it?

Reply:
We have fuzz tests for  `CreateMultiSigRedeemScript` here as well (so I do not create new import dependencies).
And executing script is somewhat perfect way to test it.